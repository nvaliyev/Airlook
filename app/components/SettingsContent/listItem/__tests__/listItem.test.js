import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import ItemsList from '..';
describe(' ItemsList', () => {
    test(' ItemsList snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(< ItemsList/>);
        expect(snapshot).toMatchSnapshot();
    });
});


