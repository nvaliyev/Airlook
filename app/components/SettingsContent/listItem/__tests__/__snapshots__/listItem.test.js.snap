// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` ItemsList  ItemsList snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#F7F9FC",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "height": 30,
      }
    }
  />
  <Themed.ListItem
    leftIcon={
      Object {
        "color": "#6f347c",
        "name": "translate",
      }
    }
    title="Language"
  />
  <View
    style={
      Object {
        "height": 30,
      }
    }
  />
  <Themed.ListItem
    leftIcon={
      Object {
        "color": "#6f347c",
        "name": "notifications",
      }
    }
    switch={
      Object {
        "onValueChange": [Function],
        "thumbColor": "#5d6fc4",
        "trackColor": Object {
          "true": "#5d6fc4",
        },
        "value": undefined,
      }
    }
    title="Notification"
  />
  <View
    style={
      Object {
        "height": 30,
      }
    }
  />
  <Themed.ListItem
    chevron={true}
    leftIcon={
      Object {
        "color": "#6f347c",
        "name": "mail",
      }
    }
    onPress={[Function]}
    title="Contact us"
  />
  <Themed.ListItem
    chevron={true}
    leftIcon={
      Object {
        "color": "#6f347c",
        "name": "star",
      }
    }
    onPress={[Function]}
    title="Rate App"
  />
  <Themed.ListItem
    chevron={true}
    leftIcon={
      Object {
        "color": "#6f347c",
        "name": "gavel",
      }
    }
    onPress={[Function]}
    title="Privacy Policy"
  />
</View>
`;
